server/aesd_ioctl.h

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
@kevintom98 you shouldn't need to duplicate this file here, can share between the driver and userspace implementation.